keyboards/keebart/sofle_choc_pro/keyboard.json

```
@@ -0,0 +1,236 @@
+            {"matrix": [3, 1], "x": 16, "y": 47, "flags": 4},
+            {"matrix": [2, 1], "x": 16, "y": 33, "flags": 4},
+            {"matrix": [1, 1], "x": 16, "y": 19, "flags": 4},
+            {"matrix": [0, 1], "x": 16, "y": 5, "flags": 4},
+            {"matrix": [0, 0], "x": 0, "y": 5, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 19, "flags": 4},
+            {"matrix": [2, 0], "x": 0, "y": 33, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 47, "flags": 4},
+            {"matrix": [9, 5], "x": 125, "y": 38, "flags": 4},
+            {"matrix": [9, 4], "x": 134, "y": 64, "flags": 4},
+            {"matrix": [9, 3], "x": 147, "y": 60, "flags": 4},
+            {"matrix": [8, 5], "x": 142, "y": 45, "flags": 4},
+            {"matrix": [7, 5], "x": 142, "y": 31, "flags": 4},
+            {"matrix": [6, 5], "x": 142, "y": 17, "flags": 4},
+            {"matrix": [5, 5], "x": 142, "y": 3, "flags": 4},
+            {"matrix": [5, 4], "x": 158, "y": 2, "flags": 4},
+            {"matrix": [6, 4], "x": 158, "y": 16, "flags": 4},
+            {"matrix": [7, 4], "x": 158, "y": 29, "flags": 4},
+            {"matrix": [8, 4], "x": 158, "y": 43, "flags": 4},
+            {"matrix": [9, 2], "x": 166, "y": 59, "flags": 4},
+            {"matrix": [9, 1], "x": 183, "y": 59, "flags": 4},
+            {"matrix": [8, 3], "x": 175, "y": 41, "flags": 4},
+            {"matrix": [7, 3], "x": 175, "y": 28, "flags": 4},
+            {"matrix": [6, 3], "x": 175, "y": 16, "flags": 4},
+            {"matrix": [5, 3], "x": 175, "y": 0, "flags": 4},
+            {"matrix": [5, 2], "x": 191, "y": 2, "flags": 4},
+            {"matrix": [6, 2], "x": 191, "y": 16, "flags": 4},
+            {"matrix": [7, 2], "x": 191, "y": 29, "flags": 4},
+            {"matrix": [8, 2], "x": 191, "y": 43, "flags": 4},
+            {"matrix": [9, 0], "x": 199, "y": 60, "flags": 4},
+            {"matrix": [8, 1], "x": 208, "y": 47, "flags": 4},
+            {"matrix": [7, 1], "x": 208, "y": 33, "flags": 4},
+            {"matrix": [6, 1], "x": 208, "y": 19, "flags": 4},
+            {"matrix": [5, 1], "x": 208, "y": 5, "flags": 4},
+            {"matrix": [5, 0], "x": 224, "y": 5, "flags": 4},
+            {"matrix": [6, 0], "x": 224, "y": 19, "flags": 4},
+            {"matrix": [7, 0], "x": 224, "y": 33, "flags": 4},
+            {"matrix": [8, 0], "x": 224, "y": 47, "flags": 4}
+        ],
+        "max_brightness": 50,
+        "sleep": true,
+        "split_count": [30, 30]
+    },
+    "split": {
+        "bootmagic": {
+            "matrix": [5, 0]
+        },
+        "enabled": true,
+        "handedness": {
+            "pin": "GP21"
+        },
+        "serial": {
+            "driver": "vendor",
+            "pin": "GP12"
+        },
+        "transport": {
+            "sync": {
+                "indicators": true,
+                "layer_state": true,
+                "matrix_state": true
+            },
+            "watchdog": true
```

Doesnt make sense with `USB_VBUS_PIN` defined
```suggestion
            }
```